app/src/main/java/com/arygm/quickfix/ui/elements/QuickFixDetailsdescription.kt

```diff
@@ -0,0 +1,157 @@
+
+        // Combine services logic
+        val includedServicesToShow = quickFix.includedServices.take(2)
+        val addOnServicesToShow = quickFix.addOnServices.take(2)
+
+        // If the total number of displayed services is less than 4, fill the remainder with the
+        // other list
+        val extraIncludedServices =
+            quickFix.includedServices
+                .drop(2)
+                .take(4 - includedServicesToShow.size - addOnServicesToShow.size)
+        val extraAddOnServices =
+            quickFix.addOnServices
+                .drop(2)
+                .take(4 - includedServicesToShow.size - addOnServicesToShow.size)
+
+        val allServicesToShow =
+            includedServicesToShow +
+                addOnServicesToShow +
+                extraIncludedServices +
+                extraAddOnServices
+
+        Column(modifier = Modifier.fillMaxWidth(), horizontalAlignment = Alignment.Start) {
+          // Images Section
+          ImageSelector(screenWidth, screenHeight * 1.2f, quickFix)
+
+          // Selected Services
+          Spacer(modifier = Modifier.height(screenHeight * 0.02f))
+
+          Text(
+              text = "Selected Services",
+              style = poppinsTypography.headlineMedium,
+              color = MaterialTheme.colorScheme.onBackground,
+              modifier = Modifier.padding(bottom = screenHeight * 0.01f))
+
+          allServicesToShow.forEach { service ->
+            Row(
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier.padding(bottom = screenHeight * 0.01f)) {
+                  Icon(
+                      imageVector =
+                          if (quickFix.includedServices.contains(service)) Icons.Default.Check
+                          else Icons.Default.Star,
+                      contentDescription = null,
+                      tint = MaterialTheme.colorScheme.primary,
+                      modifier = Modifier.size(screenWidth * 0.05f))
+                  Spacer(modifier = Modifier.width(screenWidth * 0.02f))
+                  Text(
+                      text = service.name,
+                      style = poppinsTypography.bodyMedium,
+                      color =
+                          if (quickFix.includedServices.contains(service))
+                              MaterialTheme.colorScheme.onBackground
+                          else MaterialTheme.colorScheme.primary)
+                }
+          }
+
+          // Description Section with "Show More"
+          Spacer(modifier = Modifier.height(screenHeight * 0.02f))
+          Text(
+              text = "Description",
+              style = poppinsTypography.headlineMedium,
+              color = MaterialTheme.colorScheme.onBackground,
+              modifier = Modifier.padding(bottom = screenHeight * 0.01f))
+          Column {
+            Text(
+                text = if (isExpanded) quickFix.title else quickFix.title.take(250) + "...",
```

Comment:
I just realized that we don't have an optional field in the `QuickFix` data class for description.
It won't be `quickfix.title` but rather `quickfix.description` and we should only display the Description section if the text isn't empty, but I'll fix that in my next PR 😄

Reply:
Thank you for the feedback! I'll make sure to keep this in mind for my next PR. Please feel free to point out anything else I can improve. 😊